Makefile

```diff
@@ -8,6 +8,11 @@ KUBECONFIG ?= ./openshift-files/auth/kubeconfig
 all:
 	$(error Please specify a make target)
 
+.PHONY: dependencies
```

Comment:
Any reason why dependencies would not be formal dependency of `make cluster`? That way it would automatically run.

Reply:
It would be nice to be able to run the `cluster` target without internet access. But maybe we can control that somehow?

Reply:
I see your point. LGTM